diff --git a/lib/module/Pager.android.js b/lib/module/Pager.android.js
index 2a083af5d43467317717fb13227c85b786c4218b..ca49473ca28018f56fd7f772a5754d8aa93ff3cb 100644
--- a/lib/module/Pager.android.js
+++ b/lib/module/Pager.android.js
@@ -1,4 +1,3 @@
 "use strict";
 
-export { PagerViewAdapter as Pager } from "./PagerViewAdapter.js";
-//# sourceMappingURL=Pager.android.js.map
\ No newline at end of file
+export { PanResponderAdapter as Pager } from "./PanResponderAdapter.js";
diff --git a/lib/module/Pager.ios.js b/lib/module/Pager.ios.js
index 8c01e80b785652d9777b9a8e52cc892cb7140b2c..ca49473ca28018f56fd7f772a5754d8aa93ff3cb 100644
--- a/lib/module/Pager.ios.js
+++ b/lib/module/Pager.ios.js
@@ -1,4 +1,3 @@
 "use strict";
 
-export { PagerViewAdapter as Pager } from "./PagerViewAdapter.js";
-//# sourceMappingURL=Pager.ios.js.map
\ No newline at end of file
+export { PanResponderAdapter as Pager } from "./PanResponderAdapter.js";
diff --git a/lib/module/PanResponderAdapter.js b/lib/module/PanResponderAdapter.js
index 22c170523d49834810c734aafbe4c4a5746e6637..e74563a8b65dd492e12fb32bcca7407184e1fe47 100644
--- a/lib/module/PanResponderAdapter.js
+++ b/lib/module/PanResponderAdapter.js
@@ -1,17 +1,16 @@
 "use strict";
 
 import * as React from 'react';
-import { Animated, Keyboard, PanResponder, StyleSheet, View } from 'react-native';
+import { Animated, Easing, Keyboard, PanResponder, StyleSheet, View } from 'react-native';
 import useLatestCallback from 'use-latest-callback';
 import { useAnimatedValue } from "./useAnimatedValue.js";
 import { jsx as _jsx } from "react/jsx-runtime";
 const DEAD_ZONE = 12;
 const DefaultTransitionSpec = {
-  timing: Animated.spring,
-  stiffness: 1000,
-  damping: 500,
-  mass: 3,
-  overshootClamping: true
+  timing: Animated.timing,
+  duration: 200,
+  easing: Easing.out(Easing.cubic),
+  useNativeDriver: false
 };
 export function PanResponderAdapter({
   layout,
